src/utils/components/DescriptionItem/DescriptionItemHeader.tsx

```diff
@@ -37,15 +36,17 @@ export const DescriptionItemHeader: React.FC<DescriptionItemHeaderProps> = ({
             {moreInfoURL && (
               <>
                 {t('More info: ')}
-                <Link to={moreInfoURL}>{moreInfoURL}</Link>
+                <a href={moreInfoURL}>{moreInfoURL}</a>
```

Comment:
why replacing?

Reply:
Because `Link` does not work as we would expect. It "looks" at the `history`. That's not what we want in here.
Also because everywhere where this works without issues, we already use `a` element exactly like this. It's so simple that it doesn't worth to create a new React element, IMO.